redux/reducersComp/cryptosSlice.ts

```diff
@@ -35,14 +37,39 @@ export const cryptoApiData = createAsyncThunk(
 const cryptosSlice = createSlice({
   name: 'cryptos',
   initialState,
-  reducers: {},
+  reducers: {
+    addCrypto: (state, action: PayloadAction<DataCriptoInfo | null>) => {
+      if (action.payload === null) {
+        return;
+      }
+      let check = state.cryptosData.find(
+        elem => elem.name === action.payload?.name,
+      );
+      if (check === undefined) {
+        state.cryptosData.push(action.payload);
+      } else {
+        return;
+      }
+    },
+    cleanError: state => {
+      state.error = '';
+    },
+    deleteCrypto: (state, action: PayloadAction<string>) => {
+      state.cryptosData = state.cryptosData.filter(
+        crypto => crypto.name !== action.payload,
+      );
+      removeValue(action.payload);
+    },
+  },
   extraReducers: builder => {
     builder.addCase(cryptoApiData.fulfilled, (state, action) => {
-      const check = state.cryptosData.find(
+      let check = state.cryptosData.find(
```

Comment:
why did you change it to let?

Reply:
My mistake, when i did the PR, there was some problems and i did some mixes in the code. I will let it as const again.